test/migration/setup/setup_migration_test.go

```diff
@@ -9,10 +9,6 @@ import (
 
 func TestSetupMigration(t *testing.T) {
 	// given
-	// set env var to skip the mutating webhook check on migration setup temporarily since the old deployment
-	// will deploy the webhooks with the old configuration but the tests will be expecting the new configuration
-	// This should be removed after PR https://github.com/codeready-toolchain/toolchain-e2e/pull/809 is merged
-	t.Setenv("skip-webhook-checks-on-setup", "true")
```

Comment:
in addition to Alexey's comment maybe we could keep this commented so that we can just uncomment when needed. WDYT? Of course adjusting a bit the description above.

Reply:
The webhook test was failing so I thought it was due to the migration thing. I forgot that we made the change to the wait logic. I guess there was something actually wrong with the webhook tests then.

Reply:
Yeah, I checked the failure in the other PR and it wasn't related to setup migration test suite.
Anyway, let's drop this and keep an eye on any flakiness/failures related to migration.